Pinta.Effects/Effects/GradientHelper.cs

```diff
@@ -41,8 +55,65 @@ public enum PredefinedGradients
 
 internal static class GradientHelper
 {
-	private const double DefaultMinimumValue = 0;
-	private const double DefaultMaximumValue = 1;
+	private const double DefaultStartPosition = 0;
+	private const double DefaultEndPosition = 1;
+
+	public static ColorGradient CreateBaseGradientForEffect (
+		ColorSchemeSource colorSchemeSource,
+		PredefinedGradients colorScheme,
+		RandomSeed colorSchemeSeed)
+	{
+		switch (colorSchemeSource) {
+			case ColorSchemeSource.PredefinedGradient:
+				return CreateColorGradient (colorScheme);
+			case ColorSchemeSource.SelectedColors:
+				return ColorGradient.Create (
+					PintaCore.Palette.PrimaryColor.ToColorBgra (),
```

Comment:
Rather than accessing `PintaCore.Palette`, the caller should provide an `IPaletteService`.
For unit tests, this avoids initializing a ton of things in Pinta.Core that end up failing (this is one of the reasons why the clouds effect didn't have a unit test already)